SwiftTerm/Sources/SwiftTerm/Terminal.swift

```diff
@@ -366,10 +366,10 @@ open class Terminal {
         // call this
         cols = max (options.cols, MINIMUM_COLS)
         rows = max (options.rows, MINIMUM_ROWS)
-        if isReset {
+        if buffers != nil && isReset {
```

Comment:
Curious about the reason for this change.

Reply:
`buffers` is "forced unwrapped", so it's like Optional. Here, for sake of check, buffers can't be `nil`.

The change here says: do not touch (clear) buffers unless `isReset` is set.